ReleaseNotes.md

```diff
@@ -1,5 +1,20 @@
 # Release Notes
 
+* [Fixed Issue 469][issue-469]
```

Comment:
no need of this anymore. Release notes are now automatically generated https://github.com/mojohaus/versions-maven-plugin/releases

Reply:
Done, and README updated accordingly